src/cookbook/games/achievements-leaderboard.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Add achievements and leaderboards to your mobile game
+description: >
+  How to use the games_services plugin to add functionality to your game.
+---
+
+Gamers have various motivations for playing games. In broad strokes,
+there are four major motivations: 
+[immersion, achievement, cooperation, and competition](https://meditations.metavert.io/p/game-player-motivations).
+No matter the game you build, some players want to *achieve* in it. This
+could be trophies won or secrets unlocked. Some players want to
+*compete* in it. This could be hitting high scores or accomplishing
+speedruns. These two ideas map to the concepts of *achievements* and
+*leaderboards*.
```

Comment:
No textual changes here, just breaking up the sentences so any future edits/diffs are easier to review. 

```suggestion
No matter the game you build, some players want to *achieve* in it.
This could be trophies won or secrets unlocked.
Some players want to *compete* in it.
This could be hitting high scores or accomplishing speedruns.
These two ideas map to the concepts of *achievements* and *leaderboards*.
```